backend/src/main.py

```diff
@@ -18,38 +18,67 @@
 from pipelines import processing_pipeline
 from packet import Packet
 from target import Target
+import argparse
 
 
 def main():
     """ App entry point """
-    # Error checking
-    if len(sys.argv) < 10:
-        print(len(sys.argv))
-        print("Missing File Paths")
-        exit()
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter)
 
-    # TODO: stadardize parsing of arguments
+    parser.add_argument('-t', '--target', choices=['NGT', 'APT', 'CCSG', 'CC'], default='NGT', help='The color target reference data')
+
+    """
+    Top left & bottom right of (NGT at the moment) target (pixels)
+    """
+    parser.add_argument('top_left_x', help='Pixel value of the top-left of the color target from the left end of the color-target image')
+    parser.add_argument('bottom_right_x', help='Pixel value of the bottom-right of the color target from the left end of the color-target image')
+    parser.add_argument('top_left_y', help='Pixel value of the top-left of the color target from the top end of the color-target image')
+    parser.add_argument('bottom_right_y', help='Pixel value of the bottom-right of the color target from the top end of the color-target image')
+
+    """
+    Location of white square on target image
+    """
```

Comment:
For consistency it would be better to make these # comments unless they are large multiline comments

Reply:
not a huge deal though